mux/src/pane.rs

```diff
@@ -160,6 +160,15 @@ impl LogicalLine {
         }
         (y - 1, x - idx + self.physical_lines.last().unwrap().len())
     }
+
+    pub fn is_empty(&self) -> bool {
```

Comment:
Maybe we could rename this to `is_visually_empty` ?
Also please add a small documentation line describing the function's behavior